packages/frontend/src/components/ProjectAccess/ProjectAccess.tsx

```diff
@@ -63,32 +44,82 @@ const ProjectAccess: FC<ProjectAccessProps> = ({
     const { mutate: updateAccess } =
         useUpdateProjectAccessMutation(projectUuid);
 
-    const { data: projectAccess, isInitialLoading: isProjectAccessLoading } =
-        useProjectAccess(projectUuid);
     const {
         data: organizationUsers,
         isInitialLoading: isOrganizationUsersLoading,
     } = useOrganizationUsers();
+    const { data: groups } = useOrganizationGroups(5);
```

Comment:
do we want to keep `5` here?

Reply:
@joaoviana This message is confusing because the argument is used to filter group members, not groups itself. It should be something like: `useOrganizationGroups({ includeMembers: 5 })`

I'll handle this in a separate PR!